netutil/ip_ranges.go

```diff
@@ -0,0 +1,50 @@
+package netutil
+
+import (
+	"fmt"
+	"net"
+	"strings"
+)
+
+// Default private IP ranges in CIDR notation
+const DefaultPrivateIPRanges = "10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,127.0.0.0/8,169.254.0.0/16,fc00::/7,fe80::/10"
+
+// PrivateCidrRanges holds the default private IP ranges
+var PrivateCidrRanges CidrRanges
+
+func init() {
+	// Initialize PrivateCidrRanges with default private IP ranges
+	var err error
+	PrivateCidrRanges, err = NewCidrRanges(
+		strings.Split(DefaultPrivateIPRanges, ","),
+	)
+	if err != nil {
+		panic(fmt.Errorf("failed to initialize private CIDR ranges: %w", err))
+	}
+}
+
+// CidrRanges holds a list of parsed CIDR ranges
+type CidrRanges []*net.IPNet
```

Comment:
extremely minor, 

You may want to consider implementing a `.String()` method for ease of debugging.